CHANGELOG.md

```diff
@@ -1,3 +1,34 @@
+## 2.26.0-rc.0 / 2021-03-24
+
+Notable changes:
+
+* Experimental query editor UI helps you to write correct PromQL queries, with auto-completion, syntax highlighting and linting. 
+* Native support for AWS authentication for `remote_write` to Amazon Managed Service for Prometheus. This eliminates the need for a sidecar.
```

Comment:
This is not notable, it is just a feature. The AWS service is not even GA. It feels like we promote a commercial service and I don't agree with this.